multi-container/open-policy-agent-sample/README.md

```diff
@@ -0,0 +1,50 @@
+# Running OPA as a side car in Cloud Run
+
+In this tutorial, we will use the [same example](http://openpolicyagent.org/docs/latest/http-api-authorization/) that is documented in the official OPA website, but substitute using [Cloud Run side cars](https://cloud.google.com/run/docs/deploying#sidecars) instead of Docker Compose.
+
+We'll end up with a deployment like this:
+
+![deployment diagram](cloud-run-opa-deployment-diagram.png)
+
+The goal is to use a simple HTTP web server that accepts any HTTP GET request that you issue and echoes the OPA decision back as text. OPA will fetch policy bundles from a simple bundle server. OPA, the bundle server, and the web server will be run as containers in Cloud Run. In a production environment, you likely will have a different bundle server, but it's convenient to package it together here.
+
+# Prerequsites
+
+* An up to date gcloud SDK installed
+* [opa](https://www.openpolicyagent.org/docs/latest/) installed
+* A text editor
+* a Google Cloud project with the Cloud Run API enabled
+
+# 1. Create and push a custom bundle server image
+
+* Follow [Step 1](https://www.openpolicyagent.org/docs/latest/http-api-authorization/#1-create-a-policy-bundle) in the original tutorial, resulting in creating two files called `example.rego` and `bundle.tar.gz`.
+* Create a custom bundle server with [Dockerfile.nginx](./Dockerfile.nginx) and [nginx.conf.template](./nginx.conf.template) with the following command:
+
+```bash
+docker build -f Dockerfile.nginx -t us-central1-docker.pkg.dev/<CLOUD_PROJECT>/docker/<IMAGE_NAME> .  
+
+docker push us-central1-docker.pkg.dev/<CLOUD_PROJECT>/docker/<IMAGE_NAME>
+```
+
+Note: We assume Google Cloud Artifact Registry in this tutorial but any registry accessible to Cloud Run will work
```

Comment:
Nice, do you mind moving this up the prereqs or in a section of required gcp apis like the sister sample https://github.com/GoogleCloudPlatform/cloud-run-samples/tree/main/multi-container/hello-nginx-sample#enable-required-apis?